src/status_im/transactions/styles/list_item.cljs

```diff
@@ -0,0 +1,52 @@
+(ns status-im.transactions.styles.list-item
+  (:require-macros [status-im.utils.styles :refer [defstyle]])
+  (:require [status-im.components.styles :as st]))
+
+(def item
+  {:padding-vertical   20
+   :padding-horizontal 16
+   :flex               1
+   :flex-direction     :row
+   :align-items        :center})
+
+(def item-photo
+  {:width            86
+   :height           48
+   :border-radius    100
+   :background-color st/color-dark-blue-3
```

Comment:
imo, here better to use something like common, st in styles doesn't give any additional information